Services/CrontabInstaller.php

```diff
@@ -11,6 +12,7 @@
 namespace Ecentria\Bundle\CrontabBundle\Services;
 
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
```

Comment:
This dependency has been added just to use it's constants. Consider replacing this usage by creating private constants in the service class.

Reply:
I don't think we have a good reason to redefine the same constants here. symfony/console is still direct dependency for the lib and provides runtime environment for the specific script.